docs/Reference-for-comment-syntax.md

```diff
@@ -314,6 +363,64 @@ In this sample, we can see that if the **TargetFrameworkOverride** symbol is def
   </ItemGroup>
 ```
 
+### Ignore conditions expressions in MSBuild files
```

Comment:
Is this just MSBuild, or would this be any XML. 

Is there a json way to do this?

Reply:
`-:msbuild-conditional:noEmit ` and `+:msbuild-conditional:noEmit` are to be used only for MSBuild files since they enable/disable MSBuild condition processing which is part of XML (`Condition` attribute).

`-:cnd:noEmit` and `+:cnd:noEmit` can be used with any type of files to ignore the conditions. In solution we also have an example for CSS and JSON. XML files should also work. 

Another way to document it can be probably separate article explaining all the flags available, including other files types, however it can be time consuming investigation since we have no info on it. For now I just included the cases requested in the issues. Imo these are the most common cases, as in both language and MSBuild files the syntax for template engine conditions is the same as for other condition cases:
- for language files: preprocessor directives have the same syntax
- for MSBuild: MSBuild conditions have the same syntax

For other types of files, it is less common.